src/strategies.jl

```diff
@@ -105,12 +115,13 @@ function iter_hook(strat::IterFunction, model, i)
     if mod1(i, strat.every) == strat.every
         strat.f(model, i)
     end
-    return
```

Comment:
will this change not introduce a bit of a type instability? With `return` its clear that the return type is `Void`, now its `Union{Void,WhateverStratDotFReturns}`